src/inline-help.scss

```diff
@@ -2,29 +2,18 @@
 @import "./functions";
 @import "./mixins";
 
-/*!
-.fd-tooltip
-    .fd-tooltip__content+(left, right, bottom-left, bottom-right)
-*/
 $block: #{$fd-namespace}-inline-help;
 
 .#{$block} {
   // LOCAL VARS
-  $fd-tooltip-icon-background-color: fd-color("status", 4) !default;
-  $fd-tooltip-content-background-color: fd-color("background", 2) !default;
-  $fd-tooltip-content-color: fd-color("text", 2) !default;
+  $fd-tooltip-icon-background-color: var(--sapButton_Neutral_Background) !default;
+  $fd-tooltip-content-background-color: var(--sapTile_Background) !default;
+  $fd-tooltip-content-color: var(--sapTextColor) !default;
   $fd-tooltip-padding: fd-space(xs) !default;
```

Comment:
`fd-space` is not supported. The resulted value was 8px so you can replace it with 0.5rem. 
These were the values before (in case you encounter them again in the future):
![Screen Shot 2020-05-28 at 4 16 16 PM](https://user-images.githubusercontent.com/39598672/83189063-a181d800-a0fe-11ea-8f2a-16b8d64e43ed.png)